andela/travel_tool_front
# #160370638 Delete comment functionality

#### What does this PR do?
This PR implements the delete comment functionality which enables users to delete their own comments posted on a request.

#### Description of Task to be completed?
- Write tests for delete component
- Create modal to confirm comment delete
- Fix failing tests


#### How should this be manually tested?
- Clone the repository
- Run `make start` docker build command on the terminal to install all dependencies and run the App in the docker development environment server.
- On the browser, type the localhost URL ` http://localhost:3000` and Login with your Andela email account into the App.
- On the Requests page, click on the request Id from the list of requests displayed on the request table, you should be able to see the Request details Modal displayed.
- Add a new comment using the comment box.
- On the displayed added comment, click on the `delete` button to delete the comment.
- When done, you should be able to see that the deleted comment is gone.


#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#160370638](https://www.pivotaltracker.com/story/show/160370638)


#### Screenshots (if appropriate)
<img width="963" alt="screen shot 2018-09-26 at 12 41 21 pm" src="https://user-images.githubusercontent.com/32720508/46077775-f8951400-c189-11e8-959b-32d108710b3c.png">

<img width="1440" alt="screen shot 2018-09-26 at 12 41 09 pm" src="https://user-images.githubusercontent.com/32720508/46077776-f8951400-c189-11e8-906a-3d54d99aec17.png">


#### Questions:
None
